extensions/firefox/content/pdfjschildbootstrap-enabled.js

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2014 Mozilla Foundation
```

Comment:
Yeah, like the pdfjschildbootstrap.js file, I think isn't actually used by the extension. It is a script called directly by Firefox code, in browser/components/nsBrowserGlue.js. But yeah I guess I need to do whatever packaging thing this is doing. Maybe that's how the in-tree version in Firefox gets updated.